### DISCUSSION TITLE
Progress Bar on buttons for modules

### DISCUSSION THREAD

**Discussion by cipi1965** [bitfocus/companion#2490] [Ideas]
Hello!
I was thinking about a Progress Bar feature on buttons.

For example to report:
Mixer volume
Timer progress

**Comment by dnmeid** [comment]
There is already a feature request for a progress bar for elapsed times: #2466
What you are talking about seems to be what we would call a level or gauge. This is also in the making: https://github.com/orgs/bitfocus/projects/4/views/1?pane=issue&itemId=24295037
Additionally there is a module in preparation which brings this functionality: https://github.com/thedist/companion-module-utils

**Reply by cipi1965** [reply]
Sorry, i tried to search in Issues but didn't find any results. I used the wrong terms
